src/backend/src/controllers/risks.controllers.ts

```diff
@@ -25,9 +26,11 @@ export const createRisk = async (req: Request, res: Response) => {
   const { body } = req;
   const { projectId, detail, createdById } = body;
 
-  const hasPerms = await hasRiskPermissions(createdById, projectId);
-  if (!hasPerms) {
-    return res.status(401).json({ message: 'Access Denied' });
+  const createdByUser = await prisma.user.findUnique({ where: { userId: createdById } });
+  if (createdByUser) {
```

Comment:
really should be if !createdByUser > error 404; then if role == guest > error 401 in two separate if blocks